modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -38,7 +44,7 @@
 	///the fuel amount (in seconds) that the forge has (stronger than wood)
 	var/forge_fuel_strong = 0
 	///whether the forge is capable of allowing reagent forging of the forged item.
-	//normal forges are false; to turn into true, use 3 (active) legion cores.
+	//normal forges are false; to turn into true, use 6 (active) legion cores.
```

Comment:
```suggestion
	///normal forges are false; to turn into true, use 6 (active) legion cores.
```

Reply:
the // is just an explanation, not the documentation-- which is right above it.